src/commands/update-secret.yml

```diff
@@ -0,0 +1,54 @@
+description: |
+  Updates existing secret resource on the cluster using the
+  `kubectl patch secret` command.
+  Requirements: kubeconfig should be configured to connect to the cluster.
+
+parameters:
+  secret-name:
+    description: |
+      Secret resource name
+    type: string
+  key-name:
+    description: |
+      The key name of the secret to update.
+    type: string
+  key-value:
```

Comment:
I have some concerns here but I also see some legitimate use. My main concern is we would never want a user to commit any kind of secret to their config file for obvious security reasons. At the same time, this would allow a user to use this command with the use of the new 2.0 API's pipeline parameters. 

I am leaving this one open to @lokst but this is an interesting situation/use-case we will continue to look into.